pkg/controller/operconfig/operconfig_controller.go

```diff
@@ -5,10 +5,11 @@ import (
 	"fmt"
 	"log"
 	"reflect"
-	"sigs.k8s.io/controller-runtime/pkg/cluster"
 	"strings"
 	"time"
 
+	"sigs.k8s.io/controller-runtime/pkg/cluster"
```

Comment:
remove this change, I fixed it in another PR :)

Reply:
I rebased against master, this line still exists. 
Could you link me to the PR that fixed this?